tools/ckb-auth-cli/src/main.rs

```diff
@@ -0,0 +1,84 @@
+mod auth_script;
```

Comment:
I also created a command line client for ckb-auth in https://github.com/cryptape/ckb-auth/pull/14 . We should join the force.